packages/api/src/index.ts

```diff
@@ -46,14 +51,17 @@ export default class Api implements ApiInterface {
 
   /**
    * @name signature
-   * @signature jsonrpcSignature (method: InterfaceMethodDefinition): string
+   * @signature `jsonrpcSignature (method: InterfaceMethodDefinition): string`
```

Comment:
This could be whacked. I actually saw it a day or two ago and then got ditracted. (It was for my lil jsdoc extension that built the docs on flowtype - when it still worked)

Reply:
I've pushed commit c9bc957 that removes this line